render-test/render_test.cpp

```diff
@@ -40,10 +40,14 @@ void operator delete(void* ptr, size_t) noexcept {
 #endif
 
 namespace {
-using ArgumentsTuple = std::tuple<bool, bool, uint32_t, std::string, std::vector<std::string>, std::string>;
+using ArgumentsTuple =
+    std::tuple<bool, bool, uint32_t, std::string, std::vector<std::string>, std::string, std::set<std::string>>;
 ArgumentsTuple parseArguments(int argc, char** argv) {
     args::ArgumentParser argumentParser("Mapbox GL Test Runner");
 
+    static const std::unordered_map<std::string, std::string> probeMap{
```

Comment:
eternal map or just std::map ?

Reply:
Unfortunately that is an `args` API, `args::MapFlagList` expects an unordered_map